Moq.Dapper.Test/DapperExecuteAsyncTest.cs

```diff
@@ -12,15 +15,69 @@ public void ExecuteAsync()
         {
             var connection = new Mock<DbConnection>();
 
-            connection.SetupDapperAsync(c => c.ExecuteAsync("", null, null, null, null))
+            connection.SetupDapperAsync(c => c.ExecuteAsync(It.IsAny<string>(), null, null, null, null))
                       .ReturnsAsync(1);
 
-            var result = connection.Object
-                                   .ExecuteAsync("")
-                                   .GetAwaiter()
-                                   .GetResult();
+            var actual = connection.Object
+                .ExecuteAsync("")
```

Comment:
Can you please keep the code style consistent? Indentation should match the dot from the previous line.

Reply:
The code style is now consistent